Drawsana/Shapes/Implementations/PentagonShape.swift

```diff
@@ -0,0 +1,151 @@
+//
+//  PentagonShape.swift
+//  Drawsana
+//
+//  Created by Madan Gupta on 24/12/18.
+//  Copyright © 2018 Asana. All rights reserved.
+//
+
+import CoreGraphics
+import UIKit
+
+public class PentagonShape:
```

Comment:
This is actually an Ngon, since "penta" means "five." Please rename to NgonShape. :-) 

The tool names are still correct (Pentagon and Triangle), since they define the numbers of sides.